packages/calypso-e2e/src/lib/components/editor-sidebar-block-inserter-component.ts

```diff
@@ -72,10 +72,12 @@ export class EditorSidebarBlockInserterComponent {
 		let locator;
 
 		if ( type === 'pattern' ) {
-			locator = this.editor.locator( selectors.patternResultItem( name ) );
+			locator = this.editor.locator( selectors.patternResultItem( name ) ).first();
 		} else {
-			locator = this.editor.locator( selectors.blockResultItem( name ) );
+			locator = this.editor.locator( selectors.blockResultItem( name ) ).first();
 		}
-		await locator.first().click();
+
+		await Promise.all( [ locator.hover(), locator.focus() ] );
```

Comment:
**Question:** Out of curiosity, what is this for? Was the click getting eaten somehow?

Reply:
@dpasque Yes, we had occasional failures with the `Editor: Advanced Post Flow` where the block insertion step fails due to the element click either being swallowed up or being ignored as the patterns and matching blocks load in the sidebar.

I tried the approach to wait for `networkidle` (yes, I know) to wait for the patterns to finish loading, but it appears that our patterns make a LOT of network requests so that didn't work.

I want to try hovering then focusing on the element, but fully do not expect this to be a 100% functional solution, but on the other hand I am not sure what else I can do at the moment. Perhaps blocking network requests to load the patterns while inserting the blocks then removing the intercept?

Reply:
Oh that makes sense! I mean, if this current approach works and is stable, I think it's pretty minimal and adds very little overhead! 👍  So we should go ahead with the current change!